src/Microsoft.DotNet.XHarness.TestRunners.Xunit/ThreadlessXunitTestRunner.cs

```diff
@@ -45,9 +45,13 @@ public int Run(string assemblyFileName, bool printXml, XunitFilters filters)
             var resultsXmlAssembly = new XElement("assembly");
             var resultsSink = new DelegatingXmlCreationSink(summarySink, resultsXmlAssembly);
 
+            if (Environment.GetEnvironmentVariable("XHARNESS_LOG_TEST_START") != null)
```

Comment:
this is a useful unrelated feature for diagnosing hangs in tests :)

Reply:
@akoeplinger could we update the README and add this?